book/listings/scalable_lists/2/integer_object/imp.rs

```diff
@@ -0,0 +1,69 @@
+use glib::{ParamFlags, ParamSpec, Value};
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use once_cell::sync::Lazy;
+
+use std::cell::RefCell;
+
+// Object holding the state
+#[derive(Default)]
+pub struct IntegerObject {
+    number: RefCell<i32>,
```

Comment:
```suggestion
    number: Cell<i32>,
```
for Copy types :-)

Reply:
I had that originally in the Subclassing chapter, but changed it back so I won't introduce another Cell type. Is there a good reason why I should introduce Cell within this book?